lib/process-services-cloud/src/lib/process/start-process/components/start-process-cloud.component.ts

```diff
@@ -223,10 +223,7 @@ export class StartProcessCloudComponent implements OnChanges, OnInit, OnDestroy
     private selectDefaultProcessDefinition() {
         const selectedProcess = this.getProcessDefinitionByName(this.processDefinitionName);
         if (selectedProcess) {
-            this.processDefinitionCurrent = selectedProcess;
```

Comment:
I don't think it works like that if you have a form in the default process

Reply:
The `setValue` will trigger valueChanges, which will call `selectProcessDefinitionByProcesDefinitionName` where the form will be set.